bogenliga/src/app/modules/home/components/home/home.component.ts

```diff
@@ -136,6 +138,7 @@ export class HomeComponent extends CommonComponent implements OnInit {
     }
   }
   private handleSuccessfulLogin() {
+    this.ngOnInit();
```

Comment:
Wofür den erneuten Aufruf? Damit landest du doch im worst case in einer Dauerschleife!

Reply:
Einige Sachen wurden gepusht die ich nicht pushen wollte. Sorry! Ich überarbeite alles nochmal und pushe erneut